docusaurus.config.js

```diff
@@ -36,6 +36,14 @@ const config = {
   themes: [
     "@docusaurus/theme-mermaid", // Used for diagrams
   ],
+  scripts: [
+    // Fathom analytics
+    {
+      src: "https://cdn.usefathom.com/script.js",
+      defer: true,
+      "data-site": "ZLIAETIW",
```

Comment:
env?

Reply:
@sanderpick ah, yeah, can do that. generally speaking, if the script exposes `ZLIAETIW` openly, should it still be placed in a `.env` file?
```html
<script src="https://cdn.usefathom.com/script.js" data-site="ZLIAETIW" defer></script>
```
i guess rule of thumb ==.if it's a site-specific config / var, place it in a .env...even if the info is publicly exposed?